lib/wasix/src/utils/thread_local_executor.rs

```diff
@@ -0,0 +1,96 @@
+use futures::{
```

Comment:
Why do we need this module at all? It looks like a thin wrapper that doesn't do anything extra.